obs-packaging/scripts/pkglib.sh

```diff
@@ -291,18 +291,23 @@ function pkg_version() {
 	local pkg_release="$2"
 	local commit_id="$3"
 	[ -n "${project_version}" ] || die "${FUNCNAME}: need version"
-	[ -n "${pkg_release}" ] || die "${FUNCNAME}: pkg release is needed"
 
 	pkg_version="${project_version}"
 
 	if [ -n "$commit_id" ]; then
-		pkg_version+="+git.${commit_id:0:7}"
+		# see https://github.com/openSUSE/obs-service-tar_scm/blob/master/tar_scm.service.in
+		# %h format
+		#obs short commit length is:
+		obs_short_commit_length=10
+		pkg_version+="+git.${commit_id:0:${obs_short_commit_length}}"
```

Comment:
I am not sure to understand this...why did you change from 7 to 10? `%h` is still 7 characters long.